azurerm/internal/services/dns/dns_zone_resource.go

```diff
@@ -61,13 +63,85 @@ func resourceArmDnsZone() *schema.Resource {
 				Set:      schema.HashString,
 			},
 
+			"soa_record": {
+				Type:     schema.TypeList,
+				MaxItems: 1,
+				Optional: true,
+				Computed: true,
+				Elem: &schema.Resource{
```

Comment:
Given this can't be changed/deleted can we mark it as force new?

Reply:
updated